backend/services/job.py

```
@@ -53,31 +45,106 @@ async def schedule_agent_job(payload:RunAgentRequest):
+        "conversation_history": [],
+    }
+
+
+def _build_cloud_run_client() -> run_v2.JobsClient:
+    credentials = None
+    if os.getenv("GOOGLE_CLOUD_KEY_JSON"):
+        try:
+            creds_dict = json.loads(os.getenv("GOOGLE_CLOUD_KEY_JSON"))
+            credentials = service_account.Credentials.from_service_account_info(creds_dict)
+            logger.info("Loaded Google Cloud credentials successfully.")
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)